starling/core/Starling.hx

```diff
@@ -544,10 +545,10 @@ class Starling extends EventDispatcher
         
         if (!mShareContext)
             RenderSupport._clear(mStage.color, 1.0);
-        
+                

```

Comment:
Overall I also would prefer "remove trailing whitespace" of course, but as long as we don't do it I would stick to the codebase and use 8 instead of 16 spaces.